### ISSUE TITLE
Clear and disable packages setting needs to be a default

### ISSUE THREAD

**Issue by nav9** [0x192/Universal-Android-Debloater#646]
**Describe the bug**
The "Recommended" packages have a lot of packages (like Chrome, Samsung Notes, Galaxy Store etc.) which are hard to reinstall if Universal Android Debloater (UAD) uninstalls them. The problem here is that newbies may read articles about UAD being able to restore apps that it uninstalls. Even the GUI buttons appear to indicate that an app that was uninstalled can be restored. But the reality is that the uninstall actually removes the files. This is not only confusing, also leaves people in the lurch if there's any data they didn't backup, which they didn't expect to lose by removing "Recommended" packages which they trusted that you'd have considered to actually be bloatware. For example, I'm now unable to reinstall the Samsung Store app and because of that I can't reinstall Samsung Notes either. Some minor data I had in Notes which I didn't backup, is now lost. I downloaded apk files from other sites, but the Notes data got erased when UAD uninstalled it.

**Expected behavior**
UAD has an option in Settings, which is "Clear and disable packages instead of uninstalling them". This setting needs to be activated by default. So that newbies who are trying the software for the first time do not end up uninstalling packages that are difficult to restore.

**You have a solution?**
* "Clear and disable packages instead of uninstalling them" option can be kept enabled by default. 
*  The "Restore" buttons shouldn't say "Restore" unless they are actually capable of restoring the files. If the files are already uninstalled and can't be restored, the button should be disabled instead of giving the User a false sense of security that the button can be clicked to restore the files.
* Before performing an uninstall, the User should be shown a message about whether the packages are actually going to be uninstalled or just disabled.

**Comment by Rudxain** [comment]
I agree with the issue, but I disagree with the rationale. Disabling (in UAD) also clears the data, just like an uninstall. If there's any files in user storage that were created by the app, those files will be intact (unless it's in `Android/` directory).

In newer Android versions, uninstalling an app also hides it from the app list in settings, but UAD can still fetch a list of uninstalled apps, even if UAD wasn't the one that uninstalled them.

I think `disable` should be the default, for safety reasons, and to allow the user to easily remember which apps were uninstalled

**Comment by nav9** [comment]
@Rudxain : I agree. We just need to think of it in terms of "good engineering".
